styles/globals.css

```diff
@@ -1,19 +1,63 @@
 @import '../components/Hero/hero.css';
```

Comment:
Can we use SCSS?

![image](https://user-images.githubusercontent.com/45204791/162646227-fb530e75-f5d4-4dcb-9eba-e047e8863f67.png)

https://nextjs.org/docs/basic-features/built-in-css-support

Reply:
I try work install scss whith storybook but no was possible I don how work

Reply:
it's not associated with storybook

just do 'npm install --save-dev sass' on your main project folder